### ISSUE TITLE
Getting 'Error sending email: ', <class 'AttributeError'>

### ISSUE THREAD

**Issue by jboyd170** [HackerHouseYT/Smart-Security-Camera#8]
After making the suggested edits for Python 3, I was getting an exception ('Error sending email: ', <class 'AttributeError'>).

**Comment by jboyd170** [comment]
The exception error message from sys.exc_info() returns a tuple with three values (type, value, traceback).   Since the error message in the program was only printing the first value (sys.exc_info()[0]), I didn't have enough info to debug the problem. 

I changed the exception output message to print all 3 error messages from the sys.exc_info() exception by replacing the error message line with the following: 

> except:
			print("Error sending email: ", str(sys.exc_info()))

The error message detail identified the 'Attribute Error' was related to the 'Flip' parameter from the VideoCamera object.  Not sure why the 'FLIP' parameter was creating a problem but after I removed the Flip parameter (below), the error disappeared. 

> video_camera = VideoCamera()

The program works great!  Thanks for a great project.
